httpbin/handlers.go

```diff
@@ -1013,6 +1014,20 @@ func (h *HTTPBin) Base64(w http.ResponseWriter, r *http.Request) {
 	writeResponse(w, http.StatusOK, "text/plain", result)
 }
 
+// DumpRequest - returns the given request in its HTTP/1.x wire representation.
+// The returned representation is an approximation only;
+// some details of the initial request are lost while parsing it into
+// an http.Request. In particular, the order and case of header field
+// names are lost.
+func (h *HTTPBin) DumpRequest(w http.ResponseWriter, r *http.Request) {
+	dump, err := httputil.DumpRequest(r, true)
+	if err != nil {
+		http.Error(w, fmt.Sprint(err), http.StatusInternalServerError)
```

Comment:
Nit: I'd skip `fmt.Sprint(err)` here and just write this more directly as `err.Error()`

Reply:
Already adjusted the code, thanks.